examples/objectServerExample/src/main/java/io/realm/examples/objectserver/CounterActivity.java

```diff
@@ -194,18 +186,25 @@ private void updateProgressBar(boolean downloading, boolean uploading) {
 
     private void adjustCounter(final int adjustment) {
         // A synchronized Realm can get written to at any point in time, so doing synchronous writes on the UI
-        // thread is HIGHLY discouraged as it might block longer than intended. Only use async transactions.
+        // thread is HIGHLY discouraged as it might block longer than intended. Use only async transactions.
         realm.executeTransactionAsync(new Realm.Transaction() {
             @Override
-            public void execute(Realm realm) {
+            public void execute(@Nonnull Realm realm) {
```

Comment:
Same, isn't it nonnull by default?

Reply:
Default nullability of `java.io.realm.examples.objectserver` package is not specified.
I think it's better to set default nullability by adding `package-info.java`.

Reply:
As I said above, I slightly disagree.  Only slightly, though, so I can be persuaded.  I think user code is much more likely to have per-method annotations than to do it with package-info.  ... if we do it with package info, we surely need a comment here, because, when a user copies and pastes the un-annotated version, they will get a warning and it may not be obvious how to fix it.

Reply:
If it really is true that package level annotations on interfaces are not inherited in anonymous implementations in another package, then we need to go over all our interfaces and manually add `@Nonnull`, because then I assume it will work without the user having to do it himself? Do you know @zaki50 ?